### DISCUSSION TITLE
IS it necessary to create a new maven account to create my own JARs?

### DISCUSSION THREAD

**Discussion by carlosuc3m** [bytedeco/javacpp-presets#1127] [Q&A] [answered]
Hello, I forked my the Javacpp-presets repo to create my own presets and I am trying the already existing github actions to do so. I have noticed that in order to publish the plugins to maven or nexus sonatype they use github secrets. Are those passed to the fork or do I need to create an account? And also is it good practice to publish my presets directly to nexus or should I download them and try them first?
Regards,
Carlos

**Comment by saudet** [comment, marked as answer]
You can do whatever you want with them, but since you already have a GitHub account, you might as well use GitHub Packages:
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry

**Comment by saudet** [comment]
Alternatively, you could use my account to publish artifacts by opening pull requests against special branches on this repository, similarly to how I'm doing it with @themarpe for the experimental Android support of DepthAI, see pull https://github.com/bytedeco/javacpp-presets/pull/1121.
